test-utils/devnet/src/main.rs

```diff
@@ -0,0 +1,13 @@
+extern crate node_cli;
+
+use std::path::Path;
+
+fn main() {
```

Comment:
It is okay for now, but later our devnet will be very different from the cli.
Devnet will have the tasks:
* RPC
* beacon chain block producer
* Fake consensus

CLI  will have:
* RPC
* beacon chain block producer
* beacon chain block importer
* TxFlow consensus
* network + messages_handler
* some other tasks?

Reply:
ok.. i've got a hunch that being able to inject tasks with either different functionality or no-op/passthru will be useful and ultimately lead us to less divergence between testing and production environments, and therefore less errors, but you may be right that it will be more clear without it.